src/modules/controllers.js

```diff
@@ -0,0 +1,22 @@
+const addTask = (task, arr) => {
+  const newTask = { description: task, completed: false, index: arr.length + 1 };
+  const updatedTasks = arr.concat([newTask]);
+  localStorage.setItem('tasks', JSON.stringify(updatedTasks));
+  window.location.reload();
+};
```

Comment:
- [ ] A todo can be a single word or a sentence of words in it. Currently, when adding a todo, only the first word in the multiple words is stored in the todo list. It is expected that a todo can be multiple words, for instance, "Clean the plates", but in this case, only the "Clean" is saved as the todo and the rest are nowhere to be found. Kindly work on this 👍🏽